todocore/src/main/java/com/knighttodo/todocore/rest/DayResource.java

```diff
@@ -2,24 +2,25 @@
 
 import com.knighttodo.todocore.domain.DayVO;
 import com.knighttodo.todocore.exception.CreateDayException;
+import com.knighttodo.todocore.exception.DayByDateNotFoundException;
 import com.knighttodo.todocore.exception.DayCanNotBeDeletedException;
 import com.knighttodo.todocore.exception.DayNotFoundException;
 import com.knighttodo.todocore.exception.FindAllDaysException;
-import com.knighttodo.todocore.exception.DayByDateNotFoundException;
```

Comment:
You should avoid changing the order of imports without necessity, if you use auto-format use it after you selected the part you want to format

Reply:
But if auto-format want to change order of the imports?
Maybe better to change one time?

Reply:
I don't mind, it can be an option, the code style file also can be created and usually, this should be decided by the team that owns the project. Though if you don't make changes in the file it's better to not re-arrange imports there, though I worked on a project where we selected the whole project before a commit and applied auto-format that worked well by the way.